supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -570,6 +620,58 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*:  xref:xplat[Microsoft Azure Cross-Platform Command-Line Interface]
 
+[discrete]
+[[arm64]]
+==== image:images/yes.png[yes] ARM64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term for OpenShift Container Platform (OCP) attributes, Kubernetes, Operators, APIs, and CLI objects. Use this format in general sentences when referring to OCP features.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_Creating an ARM64 compute machine set._ 
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: xref:arm64[arm64]
+
+[discrete]
+[[arm64]]
+==== image:images/yes.png[yes] arm64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, and CLI objects. Use lowercase format and backticks when referring to objects or parameters.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_Valid values are `arm64`._
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: c
+
+[discrete]
+[[bit-64-arm]]
```

Comment:
Hmm, anytime I tried to start it with the number, it was still not rendering properly. Does this look okay? Its not ideal, but wasn't sure how to get it to look correct with the number in front

Reply:
@kelbrown20 It looks ok to me in the GitHub preview.

Reply:
ID tags are hidden and are only visible to a doc user through the URL or an inspection tool, so this should not be an issue.

Reply:
Gotcha! Thank you!